lexers/m/testdata/mason_calling_component.m

```diff
@@ -0,0 +1 @@
+<& /path/to/comp.mi, name=>value &>
```

Comment:
As `re.DOTALL` is used in the python version, can we add a test case, where this is split across multiple lines?
```suggestion
<& /path/to/comp.mi,
name=>value &>
```

Reply:
Sure. Already changed `mason_calling_component.m` to support it.